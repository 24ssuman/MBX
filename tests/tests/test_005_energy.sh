#!/bin/bash

filename=$(basename "$0")
filename="${filename%.*}"

mkdir -p outputs

../../build/stage/usr/local/clusters_ultimate/bin/energy_wgrad-test inputs/${filename}.nrg > outputs/${filename}.out

