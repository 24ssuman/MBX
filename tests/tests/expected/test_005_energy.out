Energies without gradients:
system[....0]=          1.67786e+00    kcal/mol
system[....1]=          2.43401e-02    kcal/mol
system[....2]=          6.47332e+00    kcal/mol
system[....3]=         -1.73708e+01    kcal/mol
Energies with gradients:
system[....0]=          1.67786e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O                1.275e+01           1.626e+01           0.000e+00
H                6.983e+00          -1.861e+01           0.000e+00
H               -1.974e+01           2.343e+00           0.000e+00
system[....1]=          2.43401e-02    kcal/mol

Atom             GradientX           GradientY           GradientZ
O                1.445e+01           1.016e+01          -1.175e+01
H                4.015e+00          -1.026e+01           1.553e+01
H               -1.963e+01           2.318e+00          -1.814e+00
O                8.362e+00           3.816e+00           3.232e+00
H                5.892e+00          -5.749e+00          -5.167e+00
H               -1.310e+01          -2.811e-01          -3.450e-02
system[....2]=          6.47332e+00    kcal/mol

Atom             GradientX           GradientY           GradientZ
O                1.802e+01           1.621e+01          -7.452e-01
H                8.072e+00          -1.275e+01           1.272e+01
H               -2.059e+01           3.494e+00          -2.644e+00
O                5.984e+00           3.570e+00           2.004e+00
H                5.685e+00          -5.548e+00          -4.868e+00
H               -1.197e+01           1.280e+00           5.253e+00
O               -2.629e+00           4.980e+00          -2.460e+00
H                6.860e+00          -1.214e+01          -9.097e+00
H               -9.434e+00           8.992e-01          -1.658e-01
system[....3]=         -1.73708e+01    kcal/mol

Atom             GradientX           GradientY           GradientZ
F               -4.257e+00           1.478e+01           9.922e-01
O                1.919e+01           2.397e+01           5.691e+00
H                9.744e+00          -1.703e+01           8.957e+00
H               -2.165e+01           6.595e-03          -6.579e+00
O                7.807e-01           6.802e+00           4.143e+00
H                7.606e+00          -6.607e+00          -5.396e+00
H               -7.358e+00          -1.334e+00           3.108e+00
O               -1.225e+01           2.863e+01          -1.068e+00
H                9.305e+00          -1.539e+01          -9.525e+00
H               -1.108e+00          -3.383e+01          -3.225e-01
